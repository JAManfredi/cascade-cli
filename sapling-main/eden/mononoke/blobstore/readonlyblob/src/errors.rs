/*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This software may be used and distributed according to the terms of the
 * GNU General Public License version 2.
 */

use thiserror::Error;

#[derive(Debug, Error)]
pub enum ErrorKind {
    #[error("Attempt to put to ReadOnlyBlobstore for key {0}")]
    ReadOnlyPut(String),
}
