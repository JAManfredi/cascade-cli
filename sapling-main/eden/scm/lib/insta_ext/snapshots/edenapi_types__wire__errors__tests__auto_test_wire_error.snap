---
source: errors.rs
expression: "WireError::arbitrary(&mut g)"

---
{"1":"Â€","2":2786141136387072907}
